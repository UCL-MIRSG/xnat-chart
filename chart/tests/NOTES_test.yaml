templates:
  - NOTES.txt

tests:
  - it: notes should match snapshot
    set:
      web.ingress.enabled: true
    asserts:
      - matchSnapshotRaw: {}
