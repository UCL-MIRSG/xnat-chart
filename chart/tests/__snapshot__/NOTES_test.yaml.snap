notes should match snapshot:
  1: |
    |
      1. Get the application URL by running these commands:
        http://chart-example.local/
